diff --git a/upb_generator/minitable/generator.cc b/upb_generator/minitable/generator.cc
index ac7152416..4a6c5eeb8 100644
--- a/upb_generator/minitable/generator.cc
+++ b/upb_generator/minitable/generator.cc
@@ -303,25 +303,25 @@ void WriteExtension(const DefPoolPair& pools, upb::FieldDefPtr ext,
 }
 
 void RegisterExtensions(Output& output, absl::string_view unique_name) {
-  output("UPB_LINKARR_DECLARE(upb_AllExts, const upb_MiniTableExtension);\n");
-  output("UPB_CONSTRUCTOR(upb_GeneratedRegistry_Constructor, $0) {\n",
-         unique_name);
-  // TODO Although we define this function as weak and only one
-  // copy will ever exist in any binary, every instance will get registered as a
-  // separate constructor call.  To avoid duplicate registrations, we use a
-  // static variable to ensure that the function is only executed once.
-  output("  static bool finished = false;\n");
-  output("  if (finished) return;\n");
-  output("  finished = true;\n");
-  output("  static UPB_PRIVATE(upb_GeneratedExtensionListEntry) entry = {\n");
-  output("    UPB_LINKARR_START(upb_AllExts),\n");
-  output("    UPB_LINKARR_STOP(upb_AllExts),\n");
-  output("    NULL\n");
-  output("  };\n");
-  output("  UPB_ASSERT(entry.next == NULL);\n");
-  output("  entry.next = UPB_PRIVATE(upb_generated_extension_list);\n");
-  output("  UPB_PRIVATE(upb_generated_extension_list) = &entry;\n");
-  output("}\n");
+//  output("UPB_LINKARR_DECLARE(upb_AllExts, const upb_MiniTableExtension);\n");
+//  output("UPB_CONSTRUCTOR(upb_GeneratedRegistry_Constructor, $0) {\n",
+//         unique_name);
+//  // TODO Although we define this function as weak and only one
+//  // copy will ever exist in any binary, every instance will get registered as a
+//  // separate constructor call.  To avoid duplicate registrations, we use a
+//  // static variable to ensure that the function is only executed once.
+//  output("  static bool finished = false;\n");
+//  output("  if (finished) return;\n");
+//  output("  finished = true;\n");
+//  output("  static UPB_PRIVATE(upb_GeneratedExtensionListEntry) entry = {\n");
+//  output("    UPB_LINKARR_START(upb_AllExts),\n");
+//  output("    UPB_LINKARR_STOP(upb_AllExts),\n");
+//  output("    NULL\n");
+//  output("  };\n");
+//  output("  UPB_ASSERT(entry.next == NULL);\n");
+//  output("  entry.next = UPB_PRIVATE(upb_generated_extension_list);\n");
+//  output("  UPB_PRIVATE(upb_generated_extension_list) = &entry;\n");
+//  output("}\n");
 }
 
 }  // namespace
